fastlane/fastlane
# [GH Actions] Enable lock action

### Description



We are working on a bot's code migration to GitHub Actions 👉 fastlane/github-actions#3.
This PR is about enabling locking closed, inactive issues and pull requests workflow.

Enabling this workflow will not conflict with an issue-bot processing flow. 

### Testing Steps



The workflow file used for testing the action 👇

```yml
name: Locking issues and pull requests
on:
  schedule:
  - cron: "0 0 * * *"

jobs:
  lock:
    runs-on: ubuntu-latest
    steps:
    - uses: fastlane/github-actions/lock@latest
      with:
        repo-token: ${{ secrets.GITHUB_TOKEN }}
        days-before-lock: 1
```

![image](https://user-images.githubusercontent.com/10795657/81525339-a69c0480-9354-11ea-8736-ff6d57df1fa3.png)

![image](https://user-images.githubusercontent.com/10795657/81525351-b74c7a80-9354-11ea-97c2-53b31a4c1c8e.png)

> BOT_GITHUB_TOKEN is set to my personal access token, that's why you see my personal avatar and "Thanks for your contribution."  comment in a pull request comes from a different GH Action.
